Lib/test/test_io/test_general.py

```diff
@@ -2287,6 +2287,10 @@ def test_readinto(self):
                 self.assertEqual(getattr(pair, method)(data), 5)
                 self.assertEqual(bytes(data), b"abcde")
 
+        # gh-138720: C BufferedRWPair would destruct in a bad order resulting in
+        # an unraisable exception.
+        support.gc_collect()
+
```

Comment:
Is this still necessary?

Reply:
On my dev box (64 bit archlinux) adding the explicit GC from "one in a hundred" -> every time